### DISCUSSION TITLE
I don't like it

### DISCUSSION THREAD

**Discussion by Dkhitman3** [community/community#54980] [Accessibility]
### Select Topic Area

Question

### Body

Why we should give feedback to enable everything now

**Comment by ghostinhershell** [comment]
Hi @Dkhitman3, 

Thanks so much for writing in. Can you give us a bit more information about what you're referring to here? Giving a few more details might help someone give you a nudge in the right direction.

**Comment by ghostinhershell** [comment]
Hey there, 

Since it's been a while since there's been any activity in this Discussion I'm going to go ahead and close it. Please feel free to reopen it if you have any follow-up information so we can take a look. Thanks!
